hw03_pulidoc.py

```diff
@@ -0,0 +1,192 @@
+
```

Comment:
Why two files?? Pick one and delete the other. They look exactly the same.

Reply:
I believe I deleted it